komodo/check_up_to_date_pypi.py

```diff
@@ -56,7 +57,19 @@ def get_python_requirement(sources: list):
     return ""
 
 
-def compatible_versions(releases: dict, python_version):
+def is_platform_compatible(build_info: List[Dict], platform: str) -> bool:
+    if platform.lower() == "darwin":
+        platform = "macos"
```

Comment:
Why do we need this `if`? Could you explain, I do not follow

Reply:
sys.platform on mac returns darwin, but it should be macos in the filename, so I convert it to macos